.github/workflows/kind.yml

```diff
@@ -128,6 +129,69 @@ jobs:
         path: log.tar.gz
         retention-days: 30
 
+  test-e2e-encap-longruncases-only:
+    name: E2e tests with long run cases only on a Kind cluster on Linux
+    needs: [build-antrea-coverage-image, build-flow-aggregator-coverage-image]
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v1
+      with:
+        go-version: 1.15
+    - name: Download Antrea image from previous job
+      uses: actions/download-artifact@v1
+      with:
+        name: antrea-ubuntu-cov
+    - name: Download Flow Aggregator image from previous job
+      uses: actions/download-artifact@v1
+      with:
+        name: flow-aggregator-cov
+    - name: Load Antrea image
+      run:  |
+        docker load -i antrea-ubuntu-cov/antrea-ubuntu.tar
+        docker load -i flow-aggregator-cov/flow-aggregator.tar
+    - name: Install Kind
+      run: |
+        curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-$(uname)-amd64
+        chmod +x ./kind
+        sudo mv kind /usr/local/bin
+    - name: Run e2e tests
+      run: |
+        mkdir log
+        mkdir test-e2e-encap-coverage-longruncases-only
+        ANTREA_LOG_DIR=$PWD/log ANTREA_COV_DIR=$PWD/test-e2e-encap-coverage-longruncases-only ./ci/kind/test-e2e-kind.sh --encap-mode encap --coverage --longcases-only --mode-irrelevant
```

Comment:
`test-e2e-encap-without-longruncases` already tests `mode-irrelevant`. I think there's no need to test again here.

Reply:
done with new option logic